### DISCUSSION TITLE
Error when compiling for HTML5

### DISCUSSION THREAD

**Discussion by AbdooOwd** [HaxeFlixel/flixel#2529] [Q&A]
i build the game, everything is alright and when the browser get launched there is nothing in the tab, a black windows, no even HaxeFlixel intro, then i look to my console and i read this error:
`events.js:72
        throw er; // Unhandled 'error' event     
              ^
Error: listen EACCES
    at errnoException (net.js:904:11)
    at Server._listen2 (net.js:1023:19)
    at listen (net.js:1064:10)
    at net.js:1146:9
    at dns.js:72:18
    at process._tickCallback (node.js:419:13)    
    at Function.Module.runMain (module.js:499:11)
    at startup (node.js:119:16)
    at node.js:906:3
The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command lime test html5 -nolaunch --connect 6000" terminated with exit code: 1.` 

Don't tell me to look at [This](https://github.com/HaxeFlixel/flixel/discussions/2382) it does not help me.

**Comment by RichardBray** [comment]
I don't think I've ever seen this error before. Are you comfortable sharing your code so I can try and run it locally on my machine?

**Comment by Geokureli** [comment]
It's the same error as the link you gave and it honestly seems like the same problem, of running the game on a port that already has the game running. I also notice your test command has `-nolaunch` on it, doesn't that prevent the browser from launching?
